plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -377,3 +379,32 @@ export function wrapFieldResolver<TSource = any, TContext = any, TArgs = any>(
 
   return wrappedFieldResolver;
 }
+
+/**
+ * Async version of safeExecuteInTheMiddle from instrumentation package
+ * can be removed once this will be added to instrumentation package
+ * @param execute
+ * @param onFinish
+ * @param preventThrowingError
+ */
+async function safeExecuteInTheMiddleAsync<T>(
```

Comment:
nit but could we make a PR on the core to move this to the instrumentation package ?

Reply:
yea will do it, see the description :)

Reply:
Yeah i was wondering if you planned to do it, thanks for the info